Offseason2022/src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -305,6 +305,7 @@ public void setShooterMode(SHMode mode)
 
   public boolean isAtDesiredSpeed( )
   {
+    DataLogManager.log("Shooter: " + m_atDesiredSpeed);

```

Comment:
This might be a temporary DataLog message rather than long term. Still TBD on this one.